docs/deployment.md

```diff
@@ -13,52 +13,44 @@ There are orchestration templates for running Jaeger with:
   * OpenShift: [github.com/jaegertracing/jaeger-openshift](https://github.com/jaegertracing/jaeger-openshift).
 
-## Agent
+## Storage Backend
 
-Jaeger client libraries expect **jaeger-agent** process to run locally on each host.
-The agent exposes the following ports:
+Collectors require a persistent storage backend. Cassandra 3.4+ (default) and ElasticSearch are primary supported storage backends. 
 
-Port | Protocol | Function
----- | -------  | ---
-5775 | UDP      | accept zipkin.thrift over compact thrift protocol
-6831 | UDP      | accept jaeger.thrift over compact thrift protocol
-6832 | UDP      | accept jaeger.thrift over binary thrift protocol
-5778 | HTTP     | serve configs, sampling strategies
+### Cassandra
 
-It can be executed directly on the host or via Docker, as follows:
+A script is provided to initialize Cassandra keyspace and schema using Cassandra's interactive shell [`cqlsh`][cqlsh],clone the source repository following [getting started](https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING.md#getting-started) and run:
 
-```bash
-## make sure to expose only the ports you use in your deployment scenario!
-docker run \
-  --rm \
-  -p5775:5775/udp \
-  -p6831:6831/udp \
-  -p6832:6832/udp \
-  -p5778:5778/tcp \
-  jaegertracing/jaeger-agent
+```
+MODE=test sh ./plugin/storage/cassandra/schema/create.sh | cqlsh
 ```
 
-### Discovery System Integration
+Instead of starting it locally, one option is to start via Docker, so that it can be completely destroyed after use, like this:
 
-The agents can connect point to point to a single collector address, which could be
-load balanced by another infrastructure component (e.g. DNS) across multiple collectors.
-The agent can also be configured with a static list of collector addresses.
+```
+docker run --rm -it --name cassandra -p 9042:9042 cassandra
+```
 
-On Docker, a command like the following can be used:
+to create the schema. From the root of the Jaeger’s source repository,this can be executed:
 
-```bash
-docker run \
-  --rm \
-  -p5775:5775/udp \
-  -p6831:6831/udp \
-  -p6832:6832/udp \
-  -p5778:5778/tcp \
-  jaegertracing/jaeger-agent \
-  /go/bin/agent-linux --collector.host-port=jaeger-collector.jaeger-infra.svc:14267
+```
+MODE=test KEYSPACE=jaeger_v1_local ./plugin/storage/cassandra/schema/create.sh plugin/storage/cassandra/schema/v001.cql.tmpl | docker exec -i cassandra cqlsh
 ```
 
-In the future we will support different service discovery systems to dynamically load balance
-across several collectors ([issue 213](https://github.com/uber/jaeger/issues/213)).
+For production deployment, pass `MODE=prod DATACENTER={datacenter}` arguments to the script,
+where `{datacenter}` is the name used in the Cassandra configuration / network topology.
+
+The script also allows overriding TTL, keyspace name, replication factor, etc.
+Run the script without arguments to see the full list of recognized parameters.
+
+### ElasticSearch
```

Comment:
The "s" in Elasticsearch should be in lower case. Yeah, I think it's weird too :)